lms/static/certificates/sass/_components.scss

```diff
@@ -569,3 +572,89 @@
     }
 }
 
+// ------------------------------
+// #BADGES MODAL
+// ------------------------------
+.badges-overlay {
+
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+    background-color: rgba(0, 0, 0, 0.5); /*dim the background*/
+    vertical-align: middle;
+
+    .badges-modal {
+        color: #9b9b9b;
+        font-size: 14pt;
+        box-sizing: content-box;
+        right: 0;
+        left: 0;
+        max-width: 50%;
+        margin-right: auto;
+        margin-left: auto;
+        position: fixed;
+        top: 5em;
+        background: #ffffff;
+        z-index: 11;
```

Comment:
Please use a depth value from our Pattern Library's z-depths Sass map (https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Futilities%2F_variables.scss#L242). The function you would call is here - https://github.com/edx/edx-platform/blob/mattdrayer/SOL-465/static%2Fpattern-library%2Fsass%2Futilities%2F_functions.scss#L57.